test/system/helpers.bash

```diff
@@ -61,6 +61,11 @@ function basic_setup() {
         fi
     done
 
+    run_podman images --all --format '{{.Repository}}:{{.Tag}}'
```

Comment:
Can you describe (in comments, and in your commit message) why you believe this is necessary and/or a bug? I'm afraid I don't understand what this solves.

Reply:
Here is my commit message for this problem. I met it several times when the tag related tests failed. Didn't put it inside the code because it is a little hard to me to describe it in one sentence.

>  System test: Reset found_needed_image after clean up image if needed
> There are some tests that tag the desired images with new image
> name. And if these tests fail in some steps and don't clean up
> the environment. The tagged image will exist with the same id as
> the desired images. So in the setup steps, the desired images
> will be also removed as we try to remove the tagged image with
> its id. This will fail the tests right behind it. So add a check
> step after the clean up to fix it.

> Signed-off-by: Yiqiao Pu <ypu@redhat.com>

Reply:
I think I understand the problem you mention. If so, this here is not the right solution. Can you PTAL at #10008 and see if that resolves your problem?

Reply:
@ypu, #10008 has merged. Please drop this part of your commit, and rebase on current master. (A good idea anyway, since this PR is forked from a very old commit). Thank you!